<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> qm-2.0.3 builds on Mac OS X 10.3.1 but fails self-test
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:qmtest%40codesourcery.com?Subject=Re%3A%20qm-2.0.3%20builds%20on%20Mac%20OS%20X%2010.3.1%20but%20fails%20self-test&In-Reply-To=%3C2FB67DB8-1AA3-11D8-B59F-000A957CD5B0%40egenera.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000597.html">
   <LINK REL="Next"  HREF="000610.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>qm-2.0.3 builds on Mac OS X 10.3.1 but fails self-test</H1>
    <B>Steve Byan</B> 
    <A HREF="mailto:qmtest%40codesourcery.com?Subject=Re%3A%20qm-2.0.3%20builds%20on%20Mac%20OS%20X%2010.3.1%20but%20fails%20self-test&In-Reply-To=%3C2FB67DB8-1AA3-11D8-B59F-000A957CD5B0%40egenera.com%3E"
       TITLE="qm-2.0.3 builds on Mac OS X 10.3.1 but fails self-test">smb at egenera.com
       </A><BR>
    <I>Wed Nov 19 15:15:18 UTC 2003</I>
    <P><UL>
        <LI>Previous message: <A HREF="000597.html">[qmtest] Installation of qm-2.0.3-0.i386.rpm
</A></li>
        <LI>Next message: <A HREF="000610.html">[qmtest] qm-2.0.3 builds on Mac OS X 10.3.1 but fails self-test
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#598">[ date ]</a>
              <a href="thread.html#598">[ thread ]</a>
              <a href="subject.html#598">[ subject ]</a>
              <a href="author.html#598">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>A status update on QMTest on Mac OS X: I've upgraded to OS X 10.3.1  
from 10.2.8 and now QMTest 2.0.3 builds successfully.

However, it fails the self-test:

multivac% make check
cd qm &amp;&amp; PYTHONPATH=.:./zope-dtml:./zope-dtml/ExtensionClass python -O  
./setup.py -q build
package init file './test/classes/__init__.py' not found (or not a  
regular file)
package init file './test/classes/__init__.py' not found (or not a  
regular file)
cd zope-dtml/DocumentTemplate &amp;&amp;  
PYTHONPATH=.:./zope-dtml:./zope-dtml/ExtensionClass python -O  
./setup.py -q build
cd zope-dtml/ExtensionClass &amp;&amp;  
PYTHONPATH=.:./zope-dtml:./zope-dtml/ExtensionClass python -O  
./setup.py -q build
qm/test/qmtest -D tests run  \
         -c qmtest_path=qm/test/qmtest
--- TEST RESULTS  
-------------------------------------------------------------

   regress.cycle1                                : PASS

   regress.cycle2                                : PASS

   regress.cycle3                                : PASS

   regress.cycle4                                : PASS

   regress.cycle5                                : PASS

   regress.gilles1                               : PASS

   regress.noresource                            : PASS

   regress.xml1                                  : FAIL
     Child process exited unsuccessfully

     qmtest.target:
       local

     selftest.RegTest.cmdline:
       qm/test/qmtest -D
        
/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
tests/regress/xml1
       run -O
        
/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
tests/regress/xml1/results.qmr
       -o
        
/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
tests/regress/xml1/output.qmr

     selftest.RegTest.exitcode:
       256

     selftest.RegTest.stderr:


     selftest.RegTest.stdout:

       --- TEST RESULTS  
-------------------------------------------------------------

         asdfasdf                                      : ERROR   ,  
expected PASS
           An exception occurred.

           qmtest.exception:
             exceptions.OSError: [Errno 2] No such file or directory

           qmtest.target:
             local

           qmtest.traceback:

               File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/target.py&quot;, line 208, in RunTest
                 descriptor.Run(context, result)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/database.py&quot;, line 281, in Run
                 self._Execute(context, result, &quot;Run&quot;)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/database.py&quot;, line 188, in _Execute
                 eval(&quot;item.%s(context, result)&quot; % method)
                File &quot;&quot;, line 0, in ?
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/classes/command.py&quot;, line 296, in Run
                 context, result)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/classes/command.py&quot;, line 191, in RunProgram
                 exit_status = e.Run(arguments, environment, path =  
program)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/executable.py&quot;, line 242, in Run
                 raise exc_info[0], exc_info[1]


         fnord                                         : ERROR   ,  
expected PASS
           An exception occurred.

           qmtest.exception:
             exceptions.OSError: [Errno 2] No such file or directory

           qmtest.target:
             local

           qmtest.traceback:

               File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/target.py&quot;, line 208, in RunTest
                 descriptor.Run(context, result)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/database.py&quot;, line 281, in Run
                 self._Execute(context, result, &quot;Run&quot;)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/database.py&quot;, line 188, in _Execute
                 eval(&quot;item.%s(context, result)&quot; % method)
                File &quot;&quot;, line 0, in ?
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/classes/command.py&quot;, line 296, in Run
                 context, result)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/test/classes/command.py&quot;, line 191, in RunProgram
                 exit_status = e.Run(arguments, environment, path =  
program)
                File  
&quot;/Users/smb/work/projects/scsi_reservations/doc/test/QMTest/qm-2.0.3/ 
qm/executable.py&quot;, line 242, in Run
                 raise exc_info[0], exc_info[1]



       --- STATISTICS  
---------------------------------------------------------------

              2        tests total
              0 (  0%) tests as expected
              2 (100%) tests unexpected ERROR

       --- TESTS WITH UNEXPECTED OUTCOMES  
-------------------------------------------

         asdfasdf                                      : ERROR   ,  
expected PASS
           An exception occurred.

         fnord                                         : ERROR   ,  
expected PASS
           An exception occurred.



   xmldb.api.test.derived                        : PASS

   xmldb.api.test.test_interface                 : PASS

   xmldb.common.format_time1                     : PASS

   xmldb.common.graph_complete                   : PASS

   xmldb.common.graph_cycle                      : PASS

   xmldb.common.graph_pass                       : PASS

   xmldb.common.label_is_valid                   : PASS

   xmldb.common.label_thunk                      : PASS

   xmldb.common.parse_time1                      : PASS

   xmldb.common.parse_time2                      : PASS

   xmldb.common.parse_time3                      : PASS

   xmldb.common.parse_time4                      : PASS

   xmldb.common.parser_create                    : PASS

   xmldb.common.parser_dup_command_long          : PASS

   xmldb.common.parser_dup_command_short         : PASS

   xmldb.common.parser_no_long                   : PASS

   xmldb.common.parser_two_char_short            : PASS

   xmldb.test.classes.scripttest1                : PASS

   Setup xmldb.test.classes.tempdir1             : PASS

   xmldb.test.classes.tempdir1                   : PASS

   xmldb.common.parser_help                      : PASS

   xmldb.common.parser_parse                     : PASS

   Cleanup xmldb.test.classes.tempdir1           : PASS


--- STATISTICS  
---------------------------------------------------------------

       29        tests total
        1 (  3%) tests FAIL
       28 ( 97%) tests PASS

--- TESTS THAT DID NOT PASS  
--------------------------------------------------

   regress.xml1                                  : FAIL
     Child process exited unsuccessfully

make: *** [check-serial] Error 1
multivac%

--------
Steve Byan &lt;<A HREF="http://sourcerytools.com/cgi-bin/mailman/listinfo/qmtest">smb at egenera.com</A>&gt;
Software Architect
Egenera, Inc.
165 Forest Street
Marlboro, MA 01752
(508) 858-3125


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000597.html">[qmtest] Installation of qm-2.0.3-0.i386.rpm
</A></li>
	<LI>Next message: <A HREF="000610.html">[qmtest] qm-2.0.3 builds on Mac OS X 10.3.1 but fails self-test
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#598">[ date ]</a>
              <a href="thread.html#598">[ thread ]</a>
              <a href="subject.html#598">[ subject ]</a>
              <a href="author.html#598">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="http://sourcerytools.com/cgi-bin/mailman/listinfo/qmtest">More information about the qmtest
mailing list</a><br>
</body></html>
